src/lib/FileHandler.ts

```diff
@@ -0,0 +1,20 @@
+import fs = require('fs');
+
+/**
+ * Handles all File and IO operations. 
+ * Mock this class to override file change behavior from unit tests.
+ */
+export class FileHandler {
+
+    async readFile(filename: string): Promise<string> {
+        return await fs.promises.readFile(filename, 'utf-8');
+    }
+
+    async mkdirIfNotExists(dir: string): Promise<void> {
```

Comment:
Nitpick: I feel like we could get away with just calling this `mkdir`, but the current name is also fine.